### DISCUSSION TITLE
ZoomImg prop and  gatsby-image-plugin

### DISCUSSION THREAD

**Discussion by Wookie93** [rpearce/react-medium-image-zoom#401] [Q&A]
I have hard time to figure out what I am doing wrong 
I want to query two images - small and big  and than pass it to Zoom component. And it works in  50% good.
I have a big image after zoom but it always take a 100vw

I checked this thread also https://github.com/rpearce/react-medium-image-zoom/discussions/349 but it not helped 

my code looks  like this 

```
const ImageList = () => {

  const zoomImageData = (image: IGatsbyImageData) => {
    const imageData = getImage(image)
    const src = imageData?.images.fallback?.src
    const { sizes, srcSet } = imageData?.images?.sources?.[0] ?? {}

    return { src, sizes, srcSet }
  }

  return (
      <div >
        {nodes.map((item: NodeProps, index: number) => (
          <div
            key={index}
          >
            <Zoom zoomImg={{ alt: 'sdsd', ...zoomImageData(item.bigImage) }}>
              <Image
                image={item.smallImage}
                alt="image"
              />
            </Zoom>
          </div>
        ))}
      </div>
  )
}
```

so I guess src, srcSet and sizes are correct,  so why max width of big Image is not respected?

**Comment by rpearce** [comment]
@Wookie93 this is going to be difficult to help see where the disconnect is without a live example on Code Sandbox or a demo repository. Could you provide one for me?
